src/App.tsx

```diff
@@ -1,9 +1,28 @@
+import {
+  Navigate,
+  Route,
+  Routes,
+} from 'react-router-dom';
 import './App.scss';
+import { Header } from './components/Header';
+import { HomePage } from './components/HomePage';
+import { NotFoundPage } from './components/NotFoundPage';
+import { PeopleList } from './components/PeopleList';
 
 const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
+  <section className="hero is-info is-fullheight">
+    <div className="hero-head">
+      <Header />
+    </div>
+
+    <Routes>
+      <Route path="/" element={<HomePage />} />
+      <Route path="home" element={<Navigate to="/" />} />
```

Comment:
`replace`?
here is why – https://reach.tech/router/api/navigate
also – this one
![Screen Shot 2022-08-05 at 09 31 42](https://user-images.githubusercontent.com/75496546/183015874-4febb8dc-7972-4e73-a320-65a2a52a4aca.png)